testassets/FunctionalTestsWebsite/Chatter.cs

```diff
@@ -0,0 +1,71 @@
+﻿#region Copyright notice and license
```

Comment:
what's the distinction between `test`  and `testassets` directories?

Reply:
tests contains the actual test code. testassets contain the system under test, which could be test services, test clients, or other artifacts.